### PR TITLE
Feature to process the finishedPlay and finishedRecording

### PR DESCRIPTION
**Pull request by itoogii**
In this feature request, I have added the code to process the messages from the Content script in the Popup window to show the Message in the textArea.

### CODE REVIEW

**Comment by itoogii** [comment]
@agamm I couldn't test it as the finished recording message for now is not ending, as Max and Yul haven't finished the detachEventListener function yet. Can you please review my changes in the meantime and suggest if something looks off?

File: src/pages/Popup/Popup.jsx [hunk 1]
```diff
@@ -99,12 +114,28 @@ const Popup = () => {
     }
   }
 
+  // function to handle when the play ended
+  function finishedPlaying(message) {
+    //handle when play ended
+    if (buttonPlayActive) playHandler();
+    //parse the finishedPlaying message on textarea
+    setTextContent(message);
```
Comment (agamm) [inline src/pages/Popup/Popup.jsx:122]: Why do we need to add the text into the text area after playing? I would remove as we want to keep the textarea with the instructions to play again. This should only be changed after recording.

File: src/pages/Popup/Popup.jsx [hunk 2]
```diff
@@ -99,12 +114,28 @@ const Popup = () => {
     }
   }
 
+  // function to handle when the play ended
+  function finishedPlaying(message) {
+    //handle when play ended
```
Comment (agamm) [inline src/pages/Popup/Popup.jsx:119]: Make sure your comments are consistent (spacing and capital case)

**Review by agamm — CHANGES_REQUESTED** [review]
A few minor changes.

**Review by agamm — APPROVED** [review]
Great!

**Pull request merged**